src/main/java/com/example/demo/repository/BookShelfRepository.java

```diff
@@ -1,13 +1,24 @@
 package com.example.demo.repository;
 
+import com.example.demo.model.Book;
 import com.example.demo.model.BookSlot;
 
 import java.util.HashMap;
 import java.util.Map;
 
 public class BookShelfRepository {
-    private static Map<Integer, BookSlot> books = new HashMap<>();
+    private static Map<Integer, BookSlot> booksMap = new HashMap<>();
 
     private static Integer nextId = 1;
 
+    static void addBook(Book book, Long userId) {
```

Comment:
This method should has public access modifier. Because now you can't use this method in other packages

Reply:
done)